org.lflang/src/org/lflang/generator/c/CModesGenerator.java

```diff
@@ -2,8 +2,11 @@
 
 import java.util.List;
 
+import org.eclipse.xtext.xbase.lib.IterableExtensions;
```

Comment:
We've been avoiding unnecessary dependencies on Xtext/Xbase in our code generators. Could you using something from the standard Java library instead?

Reply:
Sure. I removed the usage of that extension.